hmda/Jenkinsfile

```diff
@@ -82,30 +84,24 @@ volumes: [
       }
 
       container('docker') {
+        withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: 'dockerhub',
+          usernameVariable: 'DOCKER_HUB_USER', passwordVariable: 'DOCKER_HUB_PASSWORD']]) {
           withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: 'hmda-platform-jenkins-service',
             usernameVariable: 'DTR_USER', passwordVariable: 'DTR_PASSWORD']]) {
             withCredentials([string(credentialsId: 'internal-docker-registry', variable: 'DOCKER_REGISTRY_URL')]){
-              sh "docker build --rm -t=${env.DTR_USER}/hmda-platform hmda/target/docker/stage"
               if (gitTagged || gitBranch == "master" || isDeployPR) {
+                sh "docker build --rm -t=${env.DOCKER_HUB_USER}/hmda-platform:internal hmda/target/docker/stage"
 
                 //Push to DTR
                 sh """
-                  docker tag ${env.DTR_USER}/hmda-platform:${env.DOCKER_TAG} ${DOCKER_REGISTRY_URL}/${env.DTR_USER}/hmda-platform:${env.DOCKER_TAG}
+                  docker tag ${env.DOCKER_HUB_USER}/hmda-platform:internal ${DOCKER_REGISTRY_URL}/${env.DOCKER_HUB_USER}/hmda-platform:${env.DOCKER_TAG}
```

Comment:
I think at this point it might just be easier to put `/hmda` here instead of `env.DOCKER_HUB_USER`. Since we already just have `/hmda` in the `helm` upgrade line. It will also make it more clear in our code where we are pushing out images to.

Reply:
done